README.md

```diff
@@ -90,329 +58,15 @@ and also follows [Kaldi](http://kaldi-asr.org/) style data processing, feature e
-
-
-### Setup in your cluster
-
-Change `cmd.sh` according to your cluster setup.
-If you run experiments with your local machine, please use default `cmd.sh`.
-For more information about `cmd.sh` see http://kaldi-asr.org/doc/queue.html.
-It supports Grid Engine (`queue.pl`), SLURM (`slurm.pl`), etc.
-
-
-### CTC, attention, and hybrid CTC/attention
-
-ESPnet can completely switch the mode from CTC, attention, and hybrid CTC/attention
-
-```sh
-# hybrid CTC/attention (default)
-#  --mtlalpha 0.5 and --ctc_weight 0.3 in most cases
-$ ./run.sh
-
-# CTC mode
-$ ./run.sh --mtlalpha 1.0 --ctc_weight 1.0 --recog_model model.loss.best
-
-# attention mode
-$ ./run.sh --mtlalpha 0.0 --ctc_weight 0.0
-```
-
-The CTC training mode does not output the validation accuracy, and the optimum model is selected with its loss value
-(i.e., `--recog_model model.loss.best`).
-About the effectiveness of the hybrid CTC/attention during training and recognition, see [2] and [3].
-
-## Known issues
-
-### Error due to ACS (Multiple GPUs)
-
-When using multiple GPUs, if the training freezes or lower performance than expected is observed, verify that PCI Express Access Control Services (ACS) are disabled.
-Larger discussions can be found at: [link1](https://devtalk.nvidia.com/default/topic/883054/multi-gpu-peer-to-peer-access-failing-on-tesla-k80-/?offset=26) [link2](https://www.linuxquestions.org/questions/linux-newbie-8/howto-list-all-users-in-system-380426/) [link3](https://github.com/pytorch/pytorch/issues/1637).
-To disable the PCI Express ACS follow instructions written [here](https://github.com/NVIDIA/caffe/issues/10). You need to have a ROOT user access or request to your administrator for it.
-
-### Error due to matplotlib
-
-If you have the following error (or other numpy related errors),
-```
-RuntimeError: module compiled against API version 0xc but this version of numpy is 0xb
-Exception in main training loop: numpy.core.multiarray failed to import
-Traceback (most recent call last):
-;
-:
-from . import _path, rcParams
-ImportError: numpy.core.multiarray failed to import
-```
-Then, please reinstall matplotlib with the following command:
-```sh
-$ cd egs/an4/asr1
-$ . ./path.sh
-$ pip install pip --upgrade; pip uninstall matplotlib; pip --no-cache-dir install matplotlib
-```
-
+## Usage
+See https://espnet.github.io/espnet/tutorial.html
 
 ## Docker Container
 
-go to docker/ and follow [README.md](https://github.com/espnet/espnet/tree/master/docker/README.md) instructions there.
+go to docker/ and follow https://espnet.github.io/espnet/docker.html instructions there.
```

Comment:
should not this line need to be changed to:
`go to doc/ and follow [instructions](https://espnet.github.io/espnet/docker.html) there.`

Reply:
I think it should point to docs because all contents was moved. Why do you think so?

Reply:
My only concern is that the links is showed.  So try to hide it with:  `Follow [instructions](https://espnet.github.io/espnet/docker.html) in docs.` then?

Reply:
o.k. Thanks!